yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -374,7 +374,39 @@ Internet-Draft                 YANG Semver                  October 2022
    modifier strings.  E.g., artifact version "1.2.3_non_compatible" MUST
    NOT be defined if artifact version "1.2.3" has already been defined.
 
-3.3.1.  YANG Semver with submodules
+3.3.1.  Branching Limitations with YANG Semver
+
+   YANG artifacts that use the YANG Semver revision-label scheme MUST
+   ensure that two artifacts with the same MAJOR version number and no
+   _compatible or _non_compatible modifiers are backwards compatible.
+   Therefore, certain branching schemes cannot be used with YANG Semver.
```

Comment:
We should discuss how _compatible interacts with this. It would be too bad if all bets need to be off for _compatible.

Reply:
I'll update the ticket and we can discuss.